README.md

```diff
@@ -818,6 +818,7 @@ Table of Contents
     - Free Cloud DNS for unlimited domains.
     - Free Cloud Security with Basic DDoS Protection + 5 Firewall Rules.
     - Free VoD (Video On Demand) Platform with 10 GB Storage + 50 GB Traffic.
+  * [BlobCDN](https://blobcdn.com/) - Simple CDN for generating SVG shapes on the fly, can be customized via the url parameters.
```

Comment:
not convinced I see the value in this one, bit far out of the intended audience for this list, what would one use this for?

Reply:
Sample use case:
1. Avatar placeholder/generator as a service. No need to locally store/generate avatar pic, just create an `img` html tag with src: `https://blobcdn.com/blob.svg?seed=<USER_ID_GOES_HERE>&size=30`.  Each user ID will produce different styled blob, but will be consistent.

![](https://blobcdn.com/blob.svg?seed=101&size=30) User Alice (id=101)
![](https://blobcdn.com/blob.svg?seed=102&size=30) User Bob (id=102)
![](https://blobcdn.com/blob.svg?seed=103&size=30) User Charlie (id=103)

2. "Blob" style element on web-frontend [was popular sometimes ago](https://speckyboy.com/blob-animated-backgrounds/), so instead of manually writing SVG blobs for your web's frontend background/element, just let this service generate random (or customized) one for you.

I understand your concern though, this "SVG-blob as a service" looks underwhelming, so feel free to accept/reject this PR as you see fits.